cyipopt/scipy_interface.py

```diff
@@ -35,7 +35,6 @@
 
 import cyipopt
 
-
```

Comment:
Leave this blank line here. PEP8 convention is two blank lines between imports and other code.